scripts/generate/generate-src.ts

```diff
@@ -29,18 +57,20 @@ export function generateSrc(extensionType: string, template: Template) {
   const text = file.toString();
 
   try {
-    const outDir = path.resolve(__dirname, '../../src');
-    if (!fs.existsSync(outDir)) {
-      fs.mkdirSync(outDir);
+    if (!fs.existsSync(rootDir)) {
```

Comment:
I think we should remove this whole block, no? The rootDir should always exist since it's the cloned template folder.